### DISCUSSION TITLE
Adding more logos

### DISCUSSION THREAD

**Discussion by neo-media** [Codeinwp/logo-maker#79] [General]
Is it possible to add svg's (additional logos) for the wp plugin and edit the logo maker style? Currently it loads everything from git and adding images or moderating the css is difficult.

Thought, the logo maker is incredibly nice (and swift!).

**Comment by Soare-Robert-Daniel** [comment]
Hi,

Thanks for checking out Logo Maker. Unfortunately, this project was not made be easily managed by non-developers. So all the changes must be done via code.

To add new logos, check[ this folder](https://github.com/Codeinwp/logo-maker/tree/main/src/assets/logos). For CSS, you have [this folder ](https://github.com/Codeinwp/logo-maker/tree/main/src/assets/styles).

If you are not too well versed with code, you can use ChatGPT for guidance, and who knows, but Logo Maker may be part of the training set for it 🧠
